app/services/rate_limiter.rb

```diff
@@ -140,29 +141,29 @@ def increment_to_throttled!
 
   def key
     if @user
-      "throttle:throttle:#{@user.id}:#{throttle_type}"
+      "throttle:throttle:#{@user.id}:#{rate_limit_type}"
```

Comment:
should we update the keys to remove _throttle:throttle_?

Reply:
I don't think it's worth it, because the class is actively in use and that would mean resetting all the counts while it is deployed, old and new boxes would look at different keys

maybe a little comment that's like `# still uses throttle terminology because of persisted data`